### PR TITLE
Followers, allies, clans

### PR DESCRIPTION
**Pull request by hobovsky**
Discussion: #201 
Deploy preview: [here](https://deploy-preview-217--reverent-edison-2864ea.netlify.app/concepts/clans-allies-and-followers/)

### CODE REVIEW

**Review by DonaldKellett — APPROVED** [review]

File: content/concepts/clans-allies-and-followers/followers.md
Comment (Steffan153) [inline content/concepts/clans-allies-and-followers/followers.md:33]:
```suggestion
On your profile page, you can find the `Social` tab, which contains a couple of leaderboards. One of them, titled **"Following"**, contains a list and honor ranking of all users followed by you. Another one, titled **"Followers"**, provides a similar list of your followers.
```

File: content/concepts/clans-allies-and-followers/followers.md
Comment (Steffan153) [inline content/concepts/clans-allies-and-followers/followers.md:38]:
```suggestion
When viewing the solutions of a kata, you can choose to see only those from users you are following. This is a great way to easily find solutions from warriors you respect or know personally.
```

**Review by Steffan153 — CHANGES_REQUESTED** [review]

**Review by Steffan153 — APPROVED** [review]

**Review by kazk — APPROVED** [review]

**Comment by hobovsky** [comment]
I wanted this one to wait for wording review from @DonaldKellett, but since kazk greenlighted it, I am going to merge this.

**Comment by kazk** [comment]
FYI, I focus on checking misinformation and not spelling/grammar. I was relying on @DonaldKellett for those as well :laughing: I didn't notice anything obvious, and we can always fix it later.

**Pull request merged**
